.github/workflows/dotnet-framework-build.yml

```diff
@@ -0,0 +1,86 @@
+name: Build and publish packages
+
+on:
+  workflow_call:
+    inputs:
+      semver:
+        required: true
+        type: string
+        description: "Target version for the nuget packages"
+      source_folder:
+        required: true
+        type: string
+        description: "Source folder for the codes in the repository"
+      solution_name:
+        required: true
+        type: string
+        description: "Name of the solution to be built"
+      gh_packages_url:
+        required: false

```

Comment:
Why is this false? I don't have much context here but if the packageurl is used directly in the restore step where it adds the nuget source - shouldnt it be required? 

Or does it only ever use the default one?

Reply:
It will always fall back to the default one, just stops the URL being propagated around the consuming repos